specs/www.rst

```diff
@@ -90,8 +90,8 @@ The *connection scope* information passed in ``scope`` contains:
 * ``asgi["version"]`` (*Unicode string*) -- Version of the ASGI spec.
 
 * ``asgi["spec_version"]`` (*Unicode string*) -- Version of the ASGI
-  HTTP spec this server understands; one of ``"2.0"``, ``"2.1"``, ``"2.2"`` or
-  ``"2.3"``. Optional; if missing assume ``2.0``.
+  HTTP spec this server understands; for example: ``"2.0"``, ``"2.1"``, ``"2.2"``,
+  etc. Optional; if missing assume ``"2.0"``.
```

Comment:
Why the etc? It's just those 4 values 👀 
```suggestion
 ``"2.3"``. Optional; if missing assume ``"2.0"``.
```

Reply:
Can't it also be `2.4` or `2.5` now? It's entirely possible I've misread the spec. if so I'll close this; I agree that if those are the only valid values, this change is unnecessary.

Reply:
Yes, the www spec is now up to 2.5, so I think this makes sense.